helusers/_oidc_auth_impl.py

```diff
@@ -20,6 +20,20 @@ def __init__(self, settings=None, **kwargs):
         self.settings = settings or api_token_auth_settings
         super(ApiTokenAuthentication, self).__init__(**kwargs)
 
+    @property
+    def claims_options(self):
```

Comment:
Would it be good to add a comment, for which version(s) of `drf-oidc-auth` any overrides apply? I guess this `claim_options` applies to versions >=1.0.0. The old `get_audiences` applies to versions <1.0.0. Are there any other version dependent overrides?

Reply:
Added comment about the version. Haven't encountered any other changes in overridden methods.

Reply:
How about comments in the code? At least the comment for the `get_audiences` might be a bit difficult to find from the commit message, because the method isn't introduced in the same commit.

Reply:
Added